bgpd/bgp_route.c

```diff
@@ -4478,6 +4523,11 @@ static void bgp_static_update_safi(struct bgp *bgp, struct prefix *p,
 			/* Process change. */
 			bgp_aggregate_increment(bgp, p, ri, afi, safi);
 			bgp_process(bgp, rn, afi, safi);
+
+			if (SAFI_UNICAST == safi
```

Comment:
This check seems wrong, the safi should be SAFI_MPLS_VPN here

Reply:
Yes, fixed.

Reply:
What safi is supposed to be here?  @paulzlabn I see agreement but I don't see the change here

Reply:
@donaldsharp  In bgp_static_withdraw_safi() and bgp_static_update_safi(), it should be SAFI_MPLS_VPN because it applies to routes originating in ipv4|ipv6 vpn and leaking to a unicast vrf. In bgp_static_update() and bgp_static_withdraw(), it should be SAFI_UNICAST because it applies to routes that originate in a unicast vrf and leaking to ipv4|ipv6 vpn.